src/constants/indicators.js

```diff
@@ -111,24 +109,6 @@ export default {
     format: '0[.]0%',
     source: 'Dwelling type ratio, Other, Percentage of Dwellings',
   },
-  jobsWalk: {
```

Comment:
Some of the indicators previously added are no longer present in the 'master' data file.